Makd.mak

```diff
@@ -267,6 +270,9 @@ PKGVERSION := $(shell echo $(VERSION) | \
 # http://www.gnu.org/software/make/manual/make.html#Text-Functions
 TEST_FILTER_OUT := $C/$(SRC)/%/main.d
 
+# Directories to search for files to convert using d1to2fix
+D1TO2FIX_DIRS := $C
```

Comment:
> It won't be definable via Config.mak, which is where I think the things got a bit more weird. 

Not sure I follow you here ?
Config.mak is included at line 47, so it seems to me that if I do `D1TOFIX_DIRS ?= blah blah2` in my `Config.mak`, this will be used by default unless something is provided on the command line ? That's what we do for `dmd-transitional` in applications ATM.